docs/build/identity.rs/1.1/docs/how-tos/verifiable-credentials/revocation.mdx

```diff
@@ -0,0 +1,200 @@
+if it is 0 (zero) it is not revoked.
+
+For example, with this approach the issuer adds an index to a credential in the `credentialStatus` field, such as `"5"`.
+This part of the credential might then look like this:
+
+```json
+"credentialStatus": {
+  "id": "did:iota:EvaQhPXXsJsGgxSXGhZGMCvTt63KuAFtaGThx6a5nSpw#revocation",
+  "type": "RevocationBitmap2022",
+  "revocationBitmapIndex": "5"
+},
+```
+
+The verifier uses the `id` field (`did:iota:EvaQhPXXsJsGgxSXGhZGMCvTt63KuAFtaGThx6a5nSpw#revocation`) to look up the
+service in the issuer's DID document:
+
+```json
+{
+  "id": "did:iota:EvaQhPXXsJsGgxSXGhZGMCvTt63KuAFtaGThx6a5nSpw#revocation",
+  "type": "RevocationBitmap2022",
+  "serviceEndpoint": "data:application/octet-stream;base64,ZUp5ek1tQmdZR1NBQUFFZ1ptVUFBQWZPQUlF"
+}
+```
+
+During verification, the verifier decodes the revocation bitmap embedded in the `data` url.
+This bitmap written as a bitstring looks like this: `000001`.
+Here, the 5th bit is set, which means the credential with that index is revoked,
+while all other credentials aren't revoked.
+
+### StatusList2021
+[StatusList2021](https://www.w3.org/TR/2023/WD-vc-status-list-20230427) offers similar functionalities to `RevocationBitmap2022`
+but in a more flexible way. 
+The main difference is that `StatusList2021` is completely agnostic in regards to how the issuer's status list
+is stored and fetched, as long as its location can be encoded through an URL. For instance, the status list
+can be made available over HTTP (e.g. `http://example.com/credentials/status`) or through the
+Interplanetary File System (e.g. `ipfs://QmXDWGdVBhbDoXXzKNMhJk5ejnZgxpMBVzW4EhQaHPD3Mi`).
+
+This flexibility, although it requires the issuer to manually fetch and update its status list, allows for an arbitrary number of
+entries in the status list, in contrast with `RevocationBitmap2022` where the lenght of the list is limited by the DLT's constraints.
+
+Furthermore, `StatusList2021` introduces a new credential state: _suspended_. Suspended credentials are credentials that will not be 
+accepted by a validator as _valid_, but that might become valid again in the future. Instead, _revoked_ credentials **cannot** ever
+be valid again, as the _revoked_ state is irreversible.
+
+<Tabs groupId="language" queryString>
+<TabItem value="rust" label="Rust">
+
+```rust reference
+https://github.com/iotaledger/identity.rs/blob/main/examples/1_advanced/8_status_list_2021.rs#L86-L90
+```
+
+</TabItem>
+<TabItem value="typescript-node" label="Typescript (Node.js)">
+
+```ts reference
+https://github.com/iotaledger/identity.rs/blob/main/bindings/wasm/examples/src/1_advanced/7_status_list_2021.ts#L72-L76
+```
+
+</TabItem>
+</Tabs>
+
+First, an issuer creates a credential that encodes a certain status list, specifying its purpose (either `revocation` or `suspension`)
+and the location at which it will be available (`http://example.com/credentials/status` in this case). After creation, the issuer
```

Comment:
```suggestion
and the location at which it will be available (`https://example.com/credentials/status` in this case). After creation, the issuer
```